### PR TITLE
Prettified README

### LINKED ISSUE #11: Prettify readme
The current readme is very dull and unattractive, it needs a lot of improvement.

### PR DESCRIPTION
**Pull request by saurabhyadavz**
Closes #11

### CODE REVIEW

**Review by Monal5031 — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 1]
```diff
@@ -21,3 +26,12 @@ Credentials:
 ```
 
 Refer [docs](https://github.com/NJACKWinterOfCode/cruzz/tree/master/docs) to know more about how to use or setup the project.
+
+### Author
+---
+[Monal Shadi](https://github.com/Monal5031)
```
Comment (Monal5031) [inline README.md]: Add @mohitkyadav too

**Comment by Monal5031** [comment]
@saurabhshadow why close the PR?

**Comment by saurabhyadavz** [comment]
> @saurabhshadow why close the PR?

it was showing not checked .
some travis CI check is showing cross(**X**)

**Comment by Monal5031** [comment]
Yeah its unrelated to your changes, we have an issue open to fix it,

**Review by Monal5031 — APPROVED** [review]
LGTM 👍

**Pull request merged**